op-e2e/op_geth_test.go

```diff
@@ -40,6 +42,32 @@ func TestMissingGasLimit(t *testing.T) {
 	require.Nil(t, res)
 }
 
+// TestTxGasSameAsBlockGasLimit tests that op-geth rejects transactions that attempt to use the full block gas limit.
+// The L1 Info deposit always takes gas so the effective gas limit is lower than the full block gas limit.
+func TestTxGasSameAsBlockGasLimit(t *testing.T) {
+	parallel(t)
```

Comment:
This probably becomes `InitParallel` in a moment when https://github.com/ethereum-optimism/optimism/pull/5488 merges (sorry).

Reply:
yea NP. It's a small change. I did look at using `SendTX` from that PR, but it doesn't work with the error condition here so I just put this on top of develop.